### ISSUE TITLE
Bug: Headset Window hides after launch

### ISSUE THREAD

**Issue by r57zone** [ValveSoftware/openvr#1780]
Previously, everything worked and we can use VR hmd in extended display mode.

Everything seems to be working, but the image (Headset Window is hidden?) is not displayed.

The driver to check, is a standard old example driver, with the addition of the FreeTrack rotation readout.
https://github.com/r57zone/OpenVR-OpenTrack/releases/

You can also check by building with fixes your old driver (OpenVR SDK 1.23.7) and running.
```
virtual bool IsDisplayRealDisplay() 
{
	return true;
}

virtual DriverPose_t GetPose()
{
	pose.poseIsValid = true;
	pose.result = TrackingResult_Running_OK;
	pose.deviceIsConnected = true;
}
```
(startup on the main monitor)

**Comment by AnonymousJigg** [comment]
I had also lost the Headset view for my custom driver, after yesterday's update to SteamVR 2.0.8. Maybe you could try toggling the legacy mirror to bring it back.
It's in SteamVR Hamburger menu > Developer > Debug Commands > legacy_mirror_view_toggle

**Comment by r57zone** [comment]
@AnonymousJigg This output is probably good for debugging, but not good for a VR game.

**Comment by r57zone** [comment]
Quite a few drivers have failed due to the update, it would be great to fix the problem and get this feature back. Backward compatibility is extremely important, if XInput were not backward compatible it would not have become so popular on PC.

By constantly breaking backward compatibility, some VR players lose working equipment, become disillusioned, and the community decreases.

Also by breaking backwards compatibility we are harming the environment.

@aaronleiby @charleslvalve @keithbradner @jeremyselan

**Comment by 7ep3s** [comment]
this is also breaking
https://github.com/relativty/Relativty
and
https://github.com/HadesVR/HadesVR

some help for the DIY community would be greatly appreciated

**Comment by r57zone** [comment]
As a temporary solution, you can use an previous version of SteamVR.
1. Uninstall the current SteamVR.
2. [Unzip this file](https://github.com/r57zone/VR-tracking-apps/releases/download/1/SteamVR.1.18.31.07.21.7z) to C:\Program Files\SteamVR.
3. Run "vrstartup.exe".

**Comment by 7ep3s** [comment]
restarting the VRCompositor seems to help and the window won't vanish afterwards!

`PS C:\Program Files (x86)\Steam\steamapps\common\SteamVR\bin\win64> get-process vrcompositor | stop-process -Force;.\vrcompositor.exe
PS C:\Program Files (x86)\Steam\steamapps\common\SteamVR\bin\win64>`

**Comment by aleiby** [comment]
A fix was posted in the latest beta branch (v2.2.1).
https://store.steampowered.com/news/app/250820/view/3873722126474526180

**Comment by r57zone** [comment]
The problem is fixed in 2.2.1, thank you very much Valve for hearing us and fixing it.
